sr_utilities/scripts/interactive_marker.py

```diff
@@ -91,7 +91,7 @@ def processFeedback(self, feedback):
         selected_name = feedback.marker_name
 
         # we loop through all our interactive markers.
-        for index, name in enumerate(self.object_controls):
+        for name in self.object_controls:  # pylint: disable=C0206
```

Comment:
instead disabling lints,  self.object_controls.keys() ?

Reply:
For me, `for name in self.object_controls` was nicer than `for name in self.object_controls.keys()`. Really we should operate on `for name, object in self.object_controls.items()`, but not being able to test this file I thought it better to leave the code alone if possible

Reply:
let's wait for someone else to have a look here and come up with some other ideas? :)

Reply:
55a6f2f7c523c3c3b14b1771ed976b52db5199b5

Reply:
`.keys()` still triggers `C0206`, but I've made the change here anyway (agree that it's clearer to use `.keys()` ) 7113a011792b2cdba54e7bb8845356298efb715b